pkg/operators/kubeipresolver/kubeipresolver.go

```diff
@@ -21,22 +21,9 @@
 package kubeipresolver
```

Comment:
Please update the list of gadgets using it. Or make it more generic not to need to update it every time a new gadget start using it.